src/mono/CMakeLists.txt

```diff
@@ -419,6 +419,17 @@ else()
   message(FATAL_ERROR "TARGET_ARCH='${TARGET_ARCH}' not supported.")
 endif()
 
+######################################
+# ARCH CHECKS
+######################################
+
+if(HOST_ARM OR TARGET_ARM)
```

Comment:
it looks like we set this for WASM too in mono/mono: https://github.com/mono/mono/blob/main/configure.ac#L4440

Reply:
Might be better to put this into the HOST_ARM/TARGET_ARM sections.

Reply:
@vargaz do we need it for wasm too?